exercises/anagram/introduction.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+At a garage sale, you find a lovely vintage typewriter at a bargain price!
+Excitedly, you rush home, insert a sheet of paper, and start typing away.
+However, your excitement wanes when you examine the output: all words are garbled!
+For example, it prints "liar" instead of "rail" and "pots" instead of "stop."
+Carefully, you try again, but the result is the same.
+Apparently, all keys are wired incorrectly; for instance, the "r" key outputs an "l" and vice versa.
```

Comment:
Such faulty but fixed wiring would not effectively generate anagrams. It could only work for words that consist entirely of permutation cycles. Most output would not be anagrams of anything.

I reckon a variable _delay_ would work better: you'd still get all the letters you type, just probably in a different order.

Reply:
👍 I'll update

Reply:
I've updated. Could you check again?

Reply:
Yup, this works :+1: